--- mrbayes-3.2.4/src/configure.in	2014-11-06 06:03:37.000000000 -0800
+++ patch-files/configure.in	2015-02-11 13:19:45.545877082 -0800
@@ -94,19 +94,19 @@
 
     AC_CHECK_FILE($with_beagle/include/libhmsbeagle/libhmsbeagle/beagle.h,
        [BEAGLE_CFLAGS="-I$with_beagle/include/libhmsbeagle"],
-       [AC_CHECK_FILE($with_beagle/include/libhmsbeagle-1/libhmsbeagle/beagle.h,
-        [BEAGLE_CFLAGS="-I$with_beagle/include/libhmsbeagle-1"],   
+       [AC_CHECK_FILE($with_beagle/include/libhmsbeagle-2/libhmsbeagle/beagle.h,
+        [BEAGLE_CFLAGS="-I$with_beagle/include/libhmsbeagle-2"],   
         [AC_MSG_ERROR([beagle's library include file beagle.h is not found neither in 
          ${with_beagle}/include/libhmsbeagle/libhmsbeagle/ nor in 
-         ${with_beagle}/include/libhmsbeagle-1/libhmsbeagle/]. You should either install 
+         ${with_beagle}/include/libhmsbeagle-2/libhmsbeagle/]. You should either install 
          beagle library from http://code.google.com/p/beagle-lib/ or disable use of 
          beagle library by invoking configure script with option --with-beagle=no. 
          Note: without beagle library MrBayes may run slower.)])])
     AC_CHECK_FILE($with_beagle/include/libhmsbeagle/libhmsbeagle/platform.h,,
-       [AC_CHECK_FILE($with_beagle/include/libhmsbeagle-1/libhmsbeagle/platform.h,,   
+       [AC_CHECK_FILE($with_beagle/include/libhmsbeagle-2/libhmsbeagle/platform.h,,   
         [AC_MSG_ERROR([beagle's library include file platform.h is not found neither in 
          ${with_beagle}/include/libhmsbeagle/libhmsbeagle/ nor in 
-         ${with_beagle}/include/libhmsbeagle-1/libhmsbeagle/]. You should either install 
+         ${with_beagle}/include/libhmsbeagle-2/libhmsbeagle/]. You should either install 
          beagle library from http://code.google.com/p/beagle-lib/ or disable use of 
          beagle library by invoking configure script with option --with-beagle=no. 
          Note: without beagle library MrBayes may run slower.)])])
